packages/server/test.sh

```diff
@@ -21,6 +21,13 @@ while [[ "$#" -gt 0 ]]; do
   shift
 done
 
+if ! ph_isready -h localhost -p 5432; then
```

Comment:
Would it be worthwhile, ask Roel, to move test.sh to a **scripts** folder and then extract out this function into a separate specialized script responsible only for starting the DB also stored in the **scripts** folder?

Reply:
Then also there would be a separate script for stopping the DB. See the last line: **docker rm -f unittestpg || true**